### PR TITLE
feat: slider added

### PR DESCRIPTION
**Pull request by deekshithmd**
Added slider component and tried to add some responsiveness in the website

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
looks good to merge

**Review by purohitdheeraj — APPROVED** [review]
Slider is Looking good 
Merge It :->

**Review by Debarshi95 — APPROVED** [review]
Looks good

**Review by Sahil25061999 — APPROVED** [review]
looks good just disable the overflow-x for the documentation site.

**Comment by deekshithmd** [comment]
> looks good to merge

Thank you for review

**Comment by deekshithmd** [comment]
> Slider is Looking good Merge It :->

Thank you for review

**Comment by deekshithmd** [comment]
> looks good just disable the overflow-x for the documentation site.

Yes, I did that. Thank you for review

**Comment by deekshithmd** [comment]
> Looks good

Thank you for review

File: components/slider/slider.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../style.css">
+    <title>Slider</title>
+</head>
+<body>
+    <div class="range-slider">
+        <input type="range" name="price" id="price" min="0" max="1000" value="0" onchange="rangeSlide(this.value)">
+        <i class="fa fa-inr margin-l text-sm">.</i><span class="range-value text-md text-bold">0</span>
+    </div>
+    <script src="slider.js"></script>
```
Comment (mayuresh-tech) [inline components/slider/slider.html:15]: Good code and responsive!

**Pull request merged**